.github/workflows/docker-builds.yml

```diff
@@ -104,7 +105,7 @@ jobs:
 
       - name: Build docker image
         id: build-docker-image
-        uses: pytorch/test-infra/.github/actions/calculate-docker-image@main
+        uses: pytorch/test-infra/.github/actions/calculate-docker-image@wait-for-docker-build
```

Comment:
Testing https://github.com/pytorch/test-infra/pull/6013 that waits for Docker build